### ISSUE TITLE
Users may be unfamiliar with <, > symbols

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-gui#585]
**TL;DR:** During the Extension Categories External Playtest (8/3/17), we noticed some users were not yet familiar with the less-than sign ("<", and presumably greater-than as well, ">"). Is there a way we could make its meaning clearer in Scratch, such as e.g. using the words "less than" (etc.)?

We will be going through issues with label `playtest` soon, at which point we may want to make a single issue about some of these conceptual issues - we may want to combine this issue with https://github.com/LLK/scratch-gui/issues/483.

**Example of what we observed:** One of the users had a plan for how to use the WeDo distance sensor with a physical object to trigger the project to play a sound when the distance was closer. The value with no obstruction was 100 and the values we observed when the object passed in front of the distance sensor were 10, 20, 30, in that range. Although the user seemed to understand that we needed some kind of block that involved distance being less than a value (50, 100, something like that), they had a hard time finding the relevant block. When I asked them what the "when distance < [50]" block said, they were not sure how to read the less-than symbol. When I asked them if they knew what it was, they were unfamiliar with it.

**Comment by Kenny2github** [comment]
This seems ironic - the title of the Wiki article on this block is “() is less than ()” because of technical limitations in MediaWiki titles - and now you’re considering making the actual block say that...
I kinda think that maybe this could change with your rank? As in if you’re a New Scratcher then it says “is less than” while if you’re a Scratcher then it says “<“. I don’t like the idea of making the block wider, that’s all :/

**Comment by liam4** [comment]
>I kinda think that maybe this could change with your rank? As in if you’re a New Scratcher then it says “is less than” while if you’re a Scratcher then it says “<“.

I think that blocks changing with Scratcher/New Scratcher status would be far too confusing for its worth. And what about people who are certainly familiar with the < and > operators, or have used Scratch on a previous account? It would be inconvenient and frustrating for them to have to get a change of rank just to have these blocks be more compact.

That said, I agree that "<" is *much* more compact - and usually convenient - than "is less than"; but also that there's certainly still a problem with younger people not understanding it..!

Another similar case to this issue is "%" vs. "mod"; "mod" was went with. However, "mod" isn't all that much bigger than "%", so it wasn't that big of a deal in terms of compactness.

**Comment by Kenny2github** [comment]
Yeah - the rank suggestion was just a random thought to make it easier for people who already know about it.
I guess if that’s not happening though, then I’d say just don’t change it - New Scratchers have done fine with the current state, for better or worse - let it stay that way and don’t make things less compact.

Maybe add the fact that it compares greater/less than in the help dialog tho

**Comment by mrjacobbloom** [comment]
> Another similar case to this issue is "%" vs. "mod"; "mod" was went with. However, "mod" isn't all that much bigger than "%", so it wasn't that big of a deal in terms of compactness.

"mod" isn't that intuitive either though, "remainder" is the word we learn in school :P

Deeply nested booleans would get far harder to work with if ">" changed to "greater than" and ditto for "<".

**Comment by jwzimmer** [comment]
Hi @Kenny2github, @mrjacobbloom & @liam4, just to let you know, we are not necessarily going to change anything related to how "<" and ">" are displayed - we have seen users who are otherwise basically able to use the program get stumped on those symbols, so we want to consider what kind of options we might have to improve that use case, but that doesn't mean we will definitely be making any changes (and changes might be to resources rather than to the editor itself, etc.). So I wouldn't advise you to spend too much time on this, at this point. Thanks!

**Comment by chooper100** [comment]
Alternate suggestion:
What about a "Simple English" language for the Scratch editor?
I don't know how much would actually change as most of the blocks are fairly self explanatory, but it's just another idea

**Comment by chrisgarrity** [comment]
Moved to LLK/observations#2
